blog-net-core.csproj

```diff
@@ -12,6 +12,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
I thought you was working with dotnet core 5 or 6 and I realized that its netcoreapp3 upgrade it to 6.

Reply:
Resolved